doc/about/api/index.md

```diff
@@ -8,14 +8,14 @@ Sharg adheres to [semantic versioning](https://semver.org) and provides a stable
 one major release unless otherwise noted.
 
 For details regarding API stability, please see the
-[SeqAn documentation on API stability](http://docs.seqan.de/seqan/3-master-user/about_api.html#api_stability).
+[SeqAn documentation on API stability](https://docs.seqan.de/seqan/3-master-user/about_api.html#api_stability).
 
 # ABI stability
 
 There is no ABI stability.
 
-For details regarding API stability, please see the
-[SeqAn documentation on ABI stability](http://docs.seqan.de/seqan/3-master-user/about_api.html#autotoc_md38).
+For details regarding ABI stability, please see the
+[SeqAn documentation on ABI stability](https://docs.seqan.de/seqan/3-master-user/about_api.html#autotoc_md38).
```

Comment:
The link text is about the API, I believe it should be API not ABI
```suggestion
For details regarding API stability, please see the
[SeqAn documentation on API stability](https://docs.seqan.de/seqan/3-master-user/about_api.html#autotoc_md38).
```

Reply:
We have two headings: API and ABI. This is ABI. Got that wrong in my first try and read API for both headings.